ui/pages/swaps/slippage-buttons/slippage-buttons.js

```diff
@@ -78,9 +83,14 @@ export default function SlippageButtons({
           'slippage-buttons__header--open': open,
         })}
       >
-        <div className="slippage-buttons__header-text">
+        <Text
+          data-testid="slippage-buttons-header-text"
+          variant={TextVariant.bodySmBold}
+          marginRight={2}
+          color={TextColor.primaryDefault}
+        >
```

Comment:
A quick solution if not using the ButtonBase would be making sure the `Text` is a `span` 
```suggestion
          color={TextColor.primaryDefault}
          as="span"
        >
```

Reply:
This is a good approach. Devin please make this change